express-notes-api/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const arrayJson = [];
+
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+
+app.get('/api/notes', (req, res, next) => {
+  const data = require('./data.json');
```

Comment:
You can put just one `require('./data.json')` at the top of this file and then remove the others.